vividus-tests/src/main/resources/story/integration/TableTransformers.story

```diff
@@ -151,7 +151,18 @@ Meta:
     @issueId 647
 Then `<joined>` is equal to `line 1 line 2 line 3`
 Examples:
-{transformer=FROM_EXCEL, path=/data/excel.xlsx, sheet=Sheet1, addresses=A1, column=joined, \{lineBreakReplacement|VERBATIM\}= }
+{transformer=FROM_EXCEL, path=/data/excel.xlsx, sheet=Sheet1, addresses=A1, column=joined, \{lineBreakReplacement|VERBATIM\}= }\
+
+Scenario: Verify FROM_EXCEL transformer with different types of values
```

Comment:
Looks confusing to me, the user doesn't know anything about the types, and all the data is loaded as a string?

Reply:
This is a bugfix, right?

Reply:
Also, as an option you could combine:
When I initialize scopes variable `variableName` with values:examplesTable
Then `variable` is equal to table:table